yellowbrick/classifier/learning_curve.py

```diff
@@ -0,0 +1,240 @@
+# yellowbrick.classifier.learning_curve
+# Implements a learning curve visualization for classification
+#
+# Author:   Jason Keung <jason.s.keung@gmail.com>
+# Created:  Mon May 22 09:22:00 2017 -0500
+#
+# Copyright (C) 2017 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: learning_curve.py [] jason.s.keung@gmail.com $
+
+"""
+Implementations a learning curve visualizer for classification. 
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn.model_selection import learning_curve
+from sklearn.model_selection import ShuffleSplit
+
+from yellowbrick.base import ModelVisualizer
+
+## Packages for export
+__all__ = [
+    "LearningCurveVisualizer", 
```

Comment:
Don't forget the quick method; but also you don't need this -- just get rid of it.